Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -17,9 +17,28 @@ module Install
     ].freeze
     private_constant :DYNAMIC_LINKERS
 
+    GCC_VERSION_SUFFIX = OS::LINUX_GCC_CI_VERSION.sub("\.0", "").freeze
+
+    GCC_RUNTIME_LIBS = [
```

Comment:
Consider using `%w` to avoid the `",`

Reply:
I will also change the `DYNAMIC_LINKERS` because it would also benefit from this.